src/mixins/lit-localize-lite-mixin.ts

```diff
@@ -19,23 +19,26 @@ export const litLocalizeLiteMixin = <T extends Constructor<LitElement>>(
     public connectedCallback(): void {
       super.connectedCallback();
       this._initializeLocalizeLite();
-      this.localize = computeLocalize(
-        this.constructor.prototype,
-        this.language!,
-        this.resources!
-      );
     }
 
     protected updated(changedProperties: PropertyValues) {
       super.updated(changedProperties);
+      if (changedProperties.get("translationFragment")) {
+        this._initializeLocalizeLite();
+      }
+      if (changedProperties.has("language")) {
+        document.querySelector("html")!.setAttribute("lang", this.language!);
```

Comment:
We shouldn't do this here. This mixin is added to a lot of elements, now they all update the lang tag

Reply:
As far as I know this is only used for elements that are not inside `home-assistant` like onboarding and auth with their own entrypoint?

Reply:
On authorize it would happen 3 times as there are 3 elements that use the mixin there, will look for a solution for that.